### PR TITLE
feat(browser-starfish): create basic image view

### PR DESCRIPTION
**Pull request by DominikB2014**
Sorry about the large PR 😅 This works towards https://github.com/getsentry/sentry/issues/58723
1. moved some code around to divide JS/CSS view with Image (this is where the bulk of the code changes are)
2. Creates basic image view (this still needs a bunch of work)
image
3. Rename Javascript/Stylesheets to JS/CSS/Fonts (This will probs have to revisted again)
4. Other misc changes to support the 3 changes above.

### CODE REVIEW

File: static/app/views/performance/browser/resources/imageResourceView.tsx [hunk 1]
```diff
@@ -1,5 +0,0 @@
-function ImageResourceView() {
-  return <div>Add a sample table here, with some filters</div>;
-}
-
-export default ImageResourceView;
```
Comment (DominikB2014) [inline static/app/views/performance/browser/resources/imageResourceView.tsx:5]: Moved this file to resources/imageView/index

File: static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+      name: 'Resource module - resource table',
+      query: queryConditions.join(' '),
+      version: 2,
+      orderby: '-count()',
+      dataset: DiscoverDatasets.SPANS_INDEXED,
+    },
+    pageFilters.selection
+  );
+
+  const result = useDiscoverQuery({eventView, limit: 100, location, orgSlug});
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts:55]: needs referrer?

File: static/app/views/performance/browser/resources/imageView/resourceTable.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+          renderBodyCell,
+        }}
+        location={location}
+      />
+      <Pagination pageLinks={pageLinks} />
+    </Fragment>
+  );
+}
+
+export const getActionName = (transactionOp: string) => {
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/imageView/resourceTable.tsx:104]: do we need this?

**Review by AbhiPrasad — APPROVED** [review]

File: static/app/views/performance/browser/resources/imageView/resourceTable.tsx (hunk 3 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/imageView/resourceTable.tsx:104]: So weird, must have copied and pasted that for some where, no we don't, thats interactions code lol.
good catch!

File: static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts (hunk 2 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts:55]: done!

File: static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+    },
+    pageFilters.selection
+  );
+
+  const result = useDiscoverQuery({
+    eventView,
+    limit: 100,
+    location,
+    orgSlug,
+    referrer: 'api.performance.browswer.resource.image-table',
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts:60]:
```suggestion
    referrer: 'api.performance.browser.resource.image-table',
```

Comment (DominikB2014) [reply static/app/views/performance/browser/resources/imageView/utils/useIndexedResourcesQuery.ts]: 🙏

**Pull request merged**